R/mutate.R

```diff
@@ -84,19 +86,25 @@
 #'
 #' # The difference:
 #' tidy(qq_rec, number = 1)
-step_mutate <- function(recipe, ...,
+step_mutate <- function(recipe, 
+                        ...,
+                        .pkgs = character(),
                         role = "predictor",
                         trained = FALSE,
                         inputs = NULL,
                         skip = FALSE,
                         id = rand_id("mutate")) {
+  check_character(.pkgs)
+  recipes_pkg_check(required_pkgs.step_mutate(list(.pkgs = .pkgs)))
```

Comment:
Because of the way this interfaces with S3, I think this is a situation where snapshot testing with a non-existent `.pkgs` would be nice.

Reply:
done in [test step_mutate() with .pkgs not](https://github.com/tidymodels/recipes/pull/1319/commits/43b1110e69ca1d4faea2282d4ec454c042610c9c)